test/server.js

```diff
@@ -18,6 +18,11 @@ app.use(express.json());
 
 const spans = {};
 
+
+app.get("/sdk/info", (req, res) => {
+    res.json({ version: tracer.version });
+});
+
```

Comment:
```suggestion
const tracerVersion = require('dd-trace/package.json').version
app.get("/sdk/info", (req, res) => {
    res.json({ version: tracerVersion });
});

```
this is why node.js tests are failing i think